tests/main/routes/signup-routes.test.ts

```diff
@@ -0,0 +1,24 @@
+import request from 'supertest'
+import app from '../../../src/main/config/app'
```

Comment:
Remove "../../../src/" from import, because the `src` folder is already set as the root of the project in typescript